scripts/build-hotrod-image.sh

```diff
@@ -73,9 +85,16 @@ done
 # so we do not pass flags like -b and -t.
 bash scripts/build-upload-a-docker-image.sh -l -c example-hotrod -d examples/hotrod -p "${current_platform}"
 
-# Build all-in-one image locally (-l) for integration test
-make build-all-in-one
-bash scripts/build-upload-a-docker-image.sh -l -b -c all-in-one -d cmd/all-in-one -p "${current_platform}" -t release
+# Build all-in-one (for v1) or jaeger (for v2) image locally (-l) for integration test
+if [[ "$BINARY" == "all-in-one" ]]; then
+    # Build all-in-one image for v1
+    make build-all-in-one
+    bash scripts/build-upload-a-docker-image.sh -l -b -c all-in-one -d cmd/all-in-one -p "${current_platform}" -t release
+else
+    # Build jaeger image for v2
+    make build-jaeger
```

Comment:
we should be keeping `make build-all-in-one` for both the cases, right?

Reply:
no, what you have is correct, v2 is a different binary